Formula/docker-machine-driver-vultr.rb

```diff
@@ -0,0 +1,27 @@
+class DockerMachineDriverVultr < Formula
+  desc "Docker Machine driver plugin for Vultr Cloud"
+  homepage "https://github.com/janeczku/docker-machine-vultr"
+  url "https://github.com/janeczku/docker-machine-vultr/archive/v1.2.2.tar.gz"
+  sha256 "c22e63d832aaa750c70bce5e31655a1c113ba7e75819170b31c2973e0c924e6c"
+
+  head "https://github.com/janeczku/docker-machine-vultr.git"
+
+  depends_on "go" => :build
+  depends_on "godep" => :build
+  depends_on "docker-machine" => :recommended
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/janeczku/docker-machine-vultr").install buildpath.children
+
+    cd "src/github.com/janeczku/docker-machine-vultr" do
+      system "make"
+      bin.install "build/docker-machine-driver-vultr-v1.2.2" => "docker-machine-driver-vultr"
```

Comment:
Here it's probably a good idea to replace `v1.2.2` with `#{version}` so anyone updating the formula in future doesn't get tripped up.